app/src/main/java/com/zeapo/pwdstore/PasswordFragment.kt

```diff
@@ -174,6 +174,15 @@ class PasswordFragment : Fragment(R.layout.password_recycler_view) {
                     requireStore().movePasswords(recyclerAdapter.getSelectedItems(requireContext()))
                     return false
                 }
+                R.id.menu_edit_password -> {
+                    requireStore().renameCategory(
+                        Stack<PasswordItem>().apply {
+                            recyclerAdapter.getSelectedItems(requireContext()).forEach { push(it) }
+                        })
+                    mode.finish()
+                    //TODO : what is this return used for?
```

Comment:
According to docs [here](https://developer.android.com/reference/kotlin/androidx/appcompat/view/ActionMode.Callback?hl=en#onactionitemclicked) the boolean means we want the menu item to  still react as if our callback never intercepted it.